ci/ray_ci/core.tests.yml

```diff
@@ -1,8 +1,3 @@
 flaky_tests:
-  - //python/ray/tests:test_placement_group
-  - //python/ray/tests:test_runtime_env_working_dir_3
-  - //python/ray/tests:test_state_api_log
-  - //python/ray/tests:test_client_builder
   - //python/ray/tests:test_client_library_integration
```

Comment:
ah- i actually wasn't able to find any flaky job run with this one. @rkooo567 are you aware of any?